pmpro-donations.php

```diff
@@ -42,3 +42,36 @@ function pmprodon_plugin_row_meta( $links, $file ) {
 	return $links;
 }
 add_filter( 'plugin_row_meta', 'pmprodon_plugin_row_meta', 10, 2 );
+
+
+
+/**
+ *  Function to add custom confirmation message.
+ *
+ * @param string $message The confirmation message.
+ * @param object $invoice The invoice object.
+ * @return string $message The confirmation message.
+ * @since TBD
+ */
+function pmprodon_pmpro_confirmation_message($message, $invoice) {
+	$level_id = $_REQUEST['pmpro_level'];
```

Comment:
To avoid conflicts in the long-term, we should try to get the level from the $invoice object passed into this function instead of directly from the #_REQUEST variable

Reply:
pushed a commit. While debugging, $invoice arrived null, thought it was due the donation only level what encouraged me to add a couple of fallbacks just in case. After all donation only levels had invoice as well, not sure why $invoice was false then but kept the fallback. Let me know if you think isn't correct and i remove them. Thanks for reviewing @dparker1005  :)